repo/blob/gcs/gcs_storage.go

```diff
@@ -217,22 +217,26 @@ func tokenSourceFromCredentialsFile(ctx context.Context, fn string, scopes ...st
 	if err != nil {
 		return nil, errors.Wrap(err, "error reading credentials file")
 	}
-
-	cfg, err := google.JWTConfigFromJSON(data, scopes...)
+	fmt.Println("tokenSourceFromCredentialsFile")
```

Comment:
please remove logging

Reply:
@jkowalski thank you for the quick feedback. This PR is in Draft state, hence the left-in logging and lack of tests. I'll definitely keep your feedback in my when PR is ready :)

Reply:
Logs removed and unit tests added

Reply:
@leuyentran This print statement is still here and needs to be removed.

```suggestion
```